### PR TITLE
feat(integrations): custom alert priorities

### PR DESCRIPTION
**Pull request by cathteng**
Add docs for Pagerduty custom alert severities and Opsgenie custom alert priorities.

### CODE REVIEW

**Review by Dhrumil-Sentry — APPROVED** [review]

**Review by ykamo001 — APPROVED** [review]

File: docs/product/integrations/notification-incidents/opsgenie/index.mdx [hunk 1]
```diff
@@ -34,11 +34,11 @@ If you added an integration key during installation, it'll appear on the configu
...
 
 ![Opsgenie issue alert rule action](/product/integrations/notification-incidents/opsgenie/opsgenie-issue-alert-rule.png)
 
-If you'd rather be alerted when a specific metric crosses a threshold, you can set a [metric alert](/product/alerts/alert-types/#metric-alerts) by following the same steps as above. Once you've chosen the metric you want to be alerted about, click "Add Action", then select Opsgenie from the notification types dropdown.
+If you'd rather be alerted when a specific metric crosses a threshold, you can set a [metric alert](/product/alerts/alert-types/#metric-alerts) by following the same steps as above. Once you've chosen the metric you want to be alerted about, click "Add Action", then select Opsgenie from the notification types dropdown. You can also select a custom priority for the alert, or leave it blank to use the default severity level that Sentry provides for incident statuses ("P1" for critical status and "P2" for warning status)
```
Comment (vivianyentran) [inline docs/product/integrations/notification-incidents/opsgenie/index.mdx:41]:
```suggestion
If you'd rather be alerted when a specific metric crosses a threshold, you can set a [metric alert](/product/alerts/alert-types/#metric-alerts) by following the same steps as above. Once you've chosen the metric you want to be alerted about, click "Add Action", then select Opsgenie from the notification types dropdown. You can also select a custom priority for the alert, or leave it blank to use the default severity level that Sentry provides for incident statuses ("P1" for critical status and "P2" for warning status).
```

File: docs/product/integrations/notification-incidents/pagerduty/index.mdx [hunk 2]
```diff
@@ -44,11 +44,11 @@ If you re-generate an integration key for one of your services in PagerDuty you
 
 Click "Add Alert Rule" in the integration configuration page or go to **Alerts** and click "Create Alert" to set up a new alert rule for the PagerDuty integration.
 
-In [issue alerts](/product/alerts/alert-types/#issue-alerts), select "Send a PagerDuty notification" in the actions dropdown and then select your account and service:
+In [issue alerts](/product/alerts/alert-types/#issue-alerts), select "Send a PagerDuty notification" in the actions dropdown and then select your account and service. You can also select a custom severity for alert, or select "default" to use the default severity level that Sentry provides for issue types ("critical" for fatal, "error" for error, "warning" for warning, and "info" for debug and info issues).
```
Comment (vivianyentran) [inline docs/product/integrations/notification-incidents/pagerduty/index.mdx:47]:
```suggestion
In [issue alerts](/product/alerts/alert-types/#issue-alerts), select "Send a PagerDuty notification" in the actions dropdown and then select your account and service. You can also select a custom severity for the alert, or select "default" to use the default severity levels that Sentry provides for issue types ("critical" for fatal, "error" for error, "warning" for warning, and "info" for debug and info issues).
```

File: docs/product/integrations/notification-incidents/pagerduty/index.mdx [hunk 3]
```diff
@@ -44,11 +44,11 @@ If you re-generate an integration key for one of your services in PagerDuty you
...
 
 ![PagerDuty alert rule action](/product/integrations/notification-incidents/pagerduty/pd_alert_rule.png)
 
-In [metric alerts](/product/alerts/alert-types/#metric-alerts), select the PagerDuty option in the dropdown that corresponds to your account and the select your service:
+In [metric alerts](/product/alerts/alert-types/#metric-alerts), select the PagerDuty option in the dropdown that corresponds to your account and the select your service. You can also select a custom severity for the alert, or leave it blank to use the default severity level that Sentry provides for incident statuses ("critical" for critical status and "warning" for warning status).
```
Comment (vivianyentran) [inline docs/product/integrations/notification-incidents/pagerduty/index.mdx:51]:
```suggestion
In [metric alerts](/product/alerts/alert-types/#metric-alerts), select the PagerDuty option in the dropdown that corresponds to your account and then select your service. You can also select a custom severity for the alert, or leave it blank to use the default severity level that Sentry provides for incident statuses ("critical" for critical status and "warning" for warning status).
```

Comment (vivianyentran) [reply docs/product/integrations/notification-incidents/pagerduty/index.mdx:51]: From the image below, it looks like you can also set an "error" status?

**Review by vivianyentran — APPROVED** [review]
Added a couple small edits.

File: docs/product/integrations/notification-incidents/pagerduty/index.mdx (hunk 3 above)
Comment (cathteng) [reply docs/product/integrations/notification-incidents/pagerduty/index.mdx:51]: critical and warning are the only kinds of metric alerts you can set. the pagerduty default severities just happen to align exactly with those

**Pull request merged**